packages/expo-updates/ios/EXUpdates/Update/EXUpdatesNewUpdate.m

```diff
@@ -11,49 +11,51 @@
 
 @implementation EXUpdatesNewUpdate
 
-+ (EXUpdatesUpdate *)updateWithNewManifest:(NSDictionary *)rootManifest
++ (EXUpdatesUpdate *)updateWithNewManifest:(EXUpdatesNewRawManifest *)rootManifest
                                   response:(nullable NSURLResponse *)response
                                     config:(EXUpdatesConfig *)config
                                   database:(EXUpdatesDatabase *)database
 {
-  NSDictionary *manifest = rootManifest;
-  if (manifest[@"manifest"]) {
-    manifest = manifest[@"manifest"];
+  NSDictionary *tempManifest = rootManifest.rawManifestJSON;
+  if (tempManifest[@"manifest"]) {
+    tempManifest = tempManifest[@"manifest"];
   }
```

Comment:
Instead of doing this unpacking here, should we do this in the `EXUpdatesNewRawManifest` initializer? (Also @esamelson may know if we even need to check for this nested `"manifest"` field, which was part of an older format.) At minimum I think we could use a comment to revisit this but may want to make a change in this PR.

Reply:
FWIW, just confirmed with @jkhales that we no longer need this check as the root level `"manifest"` field has been completely removed.

Reply:
Great. Let's go ahead and remove support for the `manifest` field (either in this PR or another one)? Another PR might make more sense if we need to make this change on Android, too.

Reply:
Let's do it in a single follow-up PR for both android and iOS.